### PR TITLE
new Frama-C version 19.1

### PR DESCRIPTION
**Pull request by vprevosto**
opam file for the newly released Frama-C v19.1

### CODE REVIEW

File: packages/frama-c/frama-c.19.1/opam [hunk 1]
```diff
@@ -0,0 +1,137 @@
...
+                   #'frama-c' package exists
+]
+
+messages: [
+  "Why3 can be used by the WP plug-in for running additional automatic solvers"
+    {!why3:installed}
+  "Coq can be used with the WP plug-in for proving interactively proof obligations"
+    {!coq:installed}
+  "Alt-Ergo Graphical Interface can be used by the WP plug-in"
+    {!altgr-ergo:installed & alt-ergo <= "1.30"}
```
Comment (kit-ty-kate) [inline packages/frama-c/frama-c.19.1/opam]:
```suggestion
    {!altgr-ergo:installed & alt-ergo:version <= "1.30"}
```

File: packages/frama-c/frama-c.19.1/opam [hunk 2]
```diff
@@ -0,0 +1,137 @@
...
+  "Note: the package altgr-ergo could prevent the installation of newer versions of Alt-Ergo"
+    {!altgr-ergo:installed & alt-ergo <= "1.30" & ocaml >= "4.04.0"}
```
Comment (kit-ty-kate) [inline packages/frama-c/frama-c.19.1/opam]:
```suggestion
    {!altgr-ergo:installed & alt-ergo:version <= "1.30" & ocaml:version >= "4.04.0"}
```

File: packages/frama-c/frama-c.19.1/opam [hunk 3]
```diff
@@ -0,0 +1,137 @@
...
+  "Note: the installed package altgr-ergo could prevent the installation of newer versions of Alt-Ergo"
+    {altgr-ergo:installed & ocaml >= "4.04.0"}
```
Comment (kit-ty-kate) [inline packages/frama-c/frama-c.19.1/opam]:
```suggestion
    {altgr-ergo:installed & ocaml:version >= "4.04.0"}
```

File: packages/frama-c/frama-c.19.1/opam [hunk 4]
```diff
@@ -0,0 +1,137 @@
...
+  "ocamlfind" # needed beyond build stage, used by -load-module
+  "zarith"
+  "conf-autoconf" { build }
+  ( ( "lablgtk" { >= "2.18.2" } & "conf-gnomecanvas" )
+  | ( "lablgtk3" { >= "3.0.beta4" & os!="macos" } & "lablgtk3-sourceview3" ))
+  "conf-gtksourceview"
+  ( "alt-ergo-free" | "alt-ergo" )
+  "conf-graphviz" { post }
+  "yojson"
+]
```
Comment (kit-ty-kate) [inline packages/frama-c/frama-c.19.1/opam]:
```suggestion
  "conf-time" {with-test}
]
```
`time` is needed during tests
```
#=== ERROR while compiling frama-c.19.1 =======================================#
# context              2.0.5 | linux/x86_64 | ocaml-base-compiler.4.02.3 | pinned(https://frama-c.com/download/frama-c-19.1-Potassium.tar.gz)
# path                 ~/.opam/4.02/.opam-switch/build/frama-c.19.1
# command              /usr/bin/make -j2 PTESTS_OPTS=-error-code tests
# exit-code            2
# env-file             ~/.opam/log/frama-c-16-b50035.env
# output-file          ~/.opam/log/frama-c-16-b50035.out
### output ###
# Running      ptests
# /bin/sh: 1: time: not found
# make: *** [Makefile:1388: tests] Error 127
```

**Comment by kit-ty-kate** [comment]
The revdep failure of `pilat` is unrelated

File: packages/frama-c/frama-c.19.1/opam (hunk 2 above)
Comment (mseri) [reply packages/frama-c/frama-c.19.1/opam]: @vprevosto can you also commit this?

Comment (vprevosto) [reply packages/frama-c/frama-c.19.1/opam]: Sure, sorry for having missed it the first time 😕

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**